invenio_app_rdm/records_ui/views/decorators.py

```diff
@@ -365,20 +367,169 @@ def view(**kwargs):
+    """Release self url points to RDM record.
+
+    It points to DataCite URL if the integration is enabled, otherwise it points to the HTML URL.
+    """
+    doi_url = record["links"].get("doi")
+    html_url = record["links"]["self_html"]
+    return _get_header("cite-as", doi_url or html_url)
+
+
+def _get_signposting_types(record):
+    resource_type = record["metadata"]["resource_type"]
+    props = get_vocabulary_props(
+        "resourcetypes",
+        [
+            "props.schema.org",
+        ],
+        resource_type["id"],
+    )
+    url_schema_org = props.get("schema.org")
+    return [
+        _get_header("type", url_schema_org),
+        _get_header("type", "https://schema.org/AboutPage"),
+    ]
+
+
+def _get_signposting_authors(record):
+    authors = []
+    # Limit authors to the first 10.
+    for creator in islice(record["metadata"]["creators"], 0, 10):
+        for identifier in creator["person_or_org"].get("identifiers", []):
+            if identifier["scheme"] == "orcid":
+                authors.append(
+                    _get_header(
+                        "author", "https://orcid.org/" + identifier["identifier"]
+                    )
+                )
+    return authors
+
+
+def _get_signposting_describedbys(pid_value):
+    describedbys = []
+    for export_format, val in current_app.config.get(
+        "APP_RDM_RECORD_EXPORTERS", {}
+    ).items():
+        url = export_url_for(pid_value=pid_value, export_format=export_format)
+        content_type = val["content-type"]
+        describedbys.append(_get_header("describedby", url, content_type))
+    return describedbys
+
+
+def _get_signposting_licenses(record):
+    licenses = []
+    for right in record["metadata"].get("rights", []):
+        # First try to get `props.url` from the standard licenses,
+        # then try to get the optional `link` from the custom license.
+        url = right.get("props", {}).get("url") or right.get("link")
+        if url:
+            licenses.append(_get_header("license", url))
+    return licenses
+
+
+def _get_signposting_items(files, pid_value):
+    items = []
+    # Checking if the user has access to the files.
+    if files:
+        # Limiting the iteration to 100 files maximum.
+        for file in islice(files.to_dict()["entries"], 0, 100):
+            url = download_url_for(pid_value=pid_value, filename=file["key"])
+            items.append(_get_header("item", url, file["mimetype"]))
+    return items
+
+
+def _get_signposting_collection(pid_value):
+    ui_url = record_url_for(pid_value=pid_value)
+    return _get_header("collection", ui_url, "text/html")
+
+
+def _get_signposting_describes(pid_value):
+    ui_url = record_url_for(pid_value=pid_value)
+    return _get_header("describes", ui_url, "text/html")
+
+
+def _get_signposting_linkset(pid_value):
+    api_url = record_url_for(_app="api", pid_value=pid_value)
+    return _get_header("linkset", api_url, "application/linkset+json")
```

Comment:
Note: this is required for level 2 support and was already added in a previous pull request.
Here we only include a link of the type `"application/linkset+json"`, but the docs requires to also include a link of type `"application/linkset"`.